<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditChapterFormWindow</class>
 <widget class="QWidget" name="EditChapterFormWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Chapter</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Title:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Chapter #:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="chapterNumberSpinBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>51</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="titleTextbox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sections:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>88</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>88</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewEditSectionButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>290</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit Section</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createSectionButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>290</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Section</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="priceSpinBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>62</width>
     <height>26</height>
    </rect>
   </property>
   <property name="prefix">
    <string>$</string>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
  </widget>
  <widget class="QCheckBox" name="availabilityCheckBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>89</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Available</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Price:</string>
   </property>
  </widget>
  <widget class="QListView" name="chaptersList">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>191</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
